service/README.md

```diff
@@ -0,0 +1,76 @@
+# Chronon Feature Service
```

Comment:
Should we explicitly document the endpoints available here?

Reply:
I thought about this but I expect the docs to get outdated soon. A better approach might be something like wiring up [OpenApi / Swagger annotations](https://github.com/swagger-api/swagger-core/wiki/Swagger-2.X---Annotations). Could tackle this in a follow up.